src/Tizen.NUI.Components/Tizen.NUI.Components.csproj

```diff
@@ -4,15 +4,23 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <SupportedProfiles>mobile;tv;wearable</SupportedProfiles>
+  </PropertyGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\Tizen.Log\Tizen.Log.csproj" />
     <ProjectReference Include="..\Tizen.NUI\Tizen.NUI.csproj" />
     <ProjectReference Include="..\Tizen.System.Information\Tizen.System.Information.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="res\*.png">
+    <None Update="res\nui\images\common\*.png">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="res\nui\images\1920x1080\*.png">
```

Comment:
`Tizen.NUI.Components/res/nui/images/1920x1080/component_default_slider_thumb_n.png` is necessary for this patch..?

Reply:
I just add it as an example. (it only be 0.6KB)
This is just a temporary fix, so it would be removed later. ^^;